content/docs/user-guide/external-dependencies.md

```diff
@@ -1,22 +1,23 @@
 # External Dependencies
 
 There are cases when data is so large, or its processing is organized in a way
-such that you would like to avoid moving it out of its external/remote location.
-For example from a network attached storage (NAS), processing data on HDFS,
-running [Dask](https://dask.org/) via SSH, or having a script that streams data
-from S3 to process it.
+such that its preferable to avoid moving it from its original location, even if
+it's external or remote to the project. For example: data on a network attached
+storage (NAS), processing data on HDFS, running [Dask](https://dask.org/) via
+SSH, or for a script that streams data from S3 to process it.
 
 External <abbr>dependencies</abbr> and
 [external outputs](/doc/user-guide/managing-external-data) provide ways to track
 data outside of the <abbr>project</abbr>.
 
-## How it works
+## How external dependencies work
 
 You can specify external files or directories as dependencies for your pipeline
-stages. DVC will track changes in them and reflect this in the output of
-`dvc status`.
+[stages](/doc/command-reference/run). DVC will track changes in them and reflect
+this in the output of `dvc status`.
```

Comment:
to be honest mentioning `dvc status` is strange here. The most important thing that it will for example detect this change on `dvc repro`.

Reply:
Good point. I changed these texts in dd1501bb for both external docs since describing what deps/outs are isn't a goal here.